src/constants.js

```diff
@@ -65,3 +65,5 @@ export const HATHOR_TOKEN_CONFIG = {'name': 'Hathor', 'symbol': 'HTR', 'uid': '0
  * Hathor token default index
  */
 export const HATHOR_TOKEN_INDEX = 0;
+
+export const GTM_ID = process.env.REACT_APP_GTM_ID;
```

Comment:
I don't think we should have a `GTM_ID` for testnet explorer.

How `TagManager.initialize({ gtmId: GTM_ID })` will behave in case `GTM_ID` is an empty string or null?

Side note: change the target branch to `dev`